README.md

```diff
@@ -49,14 +49,24 @@ And to declare custom paste command use:
 let g:system_copy#paste_command='xclip -sel clipboard -o'
 ```
 
+To make the original yank vim command to works with this plugin, one can add the following to
+it's `.vimrc`:
+
+```
+" Mapping y to use copy
+nmap y <Plug>SystemCopy
+" Mapping p to use paste
+nmap p <Plug>SystemPaste
+```
+
 Installation
 ------------
 
 If you don't have a preferred installation method, I recommend using [Vundle](https://github.com/VundleVim/Vundle.vim).
 Assuming you have Vundle installed and configured, the following steps will
 install the plugin:
 
-Add the following line to your `~/.vimrc` and then run `:PluginInstall` from
+Add the following line to your `.vimrc` and then run `:PluginInstall` from
```

Comment:
Not sure why you made the change here, but `~/.vimrc` was very much intended as I try to be as explicit as possible with OSS and READMEs. `~/.vimrc` is the full path, whereas `.vimrc` is a relative path that won't work unless you're in the home directory. Mind removing this as well?

Reply:
Sure! I simply put that way because someone could have a custom `.vimrc` path. I'll fix that!